static/js/pages/independent-expenditures.js

```diff
@@ -66,6 +66,7 @@ $(document).ready(function() {
   new tables.DataTable($table, {
     title: 'Independent expenditure',
     path: 'schedules/schedule_e',
+    query: {is_notice: 'false'},
```

Comment:
If we're setting it here, do we even need to add it to the URLs from the aggregate links above?